sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/autorest.md

```diff
@@ -0,0 +1,108 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: SecurityInsights
+namespace: Azure.ResourceManager.SecurityInsights
+require: https://github.com/Azure/azure-rest-api-specs/blob/53b1affe357b3bfbb53721d0a2002382a046d3b0/specification/securityinsights/resource-manager/readme.md
+tag: package-preview-2022-08
```

Comment:
Let's remove the tag in autorest.md if it's the default tag in swagger readme.md

Reply:
As we talked, script will do the change